docs/docs/docker-images.md

```diff
@@ -8,7 +8,8 @@ For every Fetchd release a corresponding docker images is released. The full lis
 
 Current Versions:
 
-* [0.5.2, 0.5](https://github.com/fetchai/fetchd/blob/v0.5.2/Dockerfile) (used for beaconworld test networks)
+* [0.7.2, 0.7](https://github.com/fetchai/fetchd/blob/v0.7.2/Dockerfile) (used for fetchhub mainnetv2 networks)
```

Comment:
We need a piece of automation to update this on docker hub. If you send me your dockerhub id I will make sure that you have the correct permissions